_specs/ecip-1043.md

```diff
@@ -37,5 +94,13 @@ The following clients implemented ECIP
 - [] Parity
 - [] Mantis
 
+The following pools have indicated ECIP support
```

Comment:
The actual protocol governance should not be part of the specification document.

Reply:
Let's remove this section.